datacenter/dtr/2.2/guides/admin/upgrade/index.md

```diff
@@ -4,34 +4,61 @@ keywords: docker, dtr, upgrade, install
 title: Upgrade DTR
 ---
 
-The first step in upgrading to a new minor version or patch release of DTR 2.2,
-is ensuring you're running DTR 2.1. If that's not the case, start by upgrading
-your installation to version 2.1, and then upgrade to 2.2.
+DTR uses [semantic versioning](http://semver.org/) and we aim to achieve specific
+guarantees while upgrading between versions. We never support downgrading. We
+support upgrades according to the following rules:
 
-There is no downtime when upgrading a highly-available DTR cluster. If your
-DTR deployment has a single replica, schedule the upgrade to take place outside
-business peak hours to ensure the impact on your business is close to none.
+* When upgrading from one patch version to another you can skip patch versions
```

Comment:
To make it a bit more organized maybe we can put this in a table like

|From| To| Description| Supported| 
|:----|:---|:------------|----------|
|x.x.0| x.x.1| From one patch to the one after| yes|
|x.x.0| x.x.5| Skipping patches| yes|
|x.0.x| x.1.x| From one minor version to the next| yes|
|x.0.x| x.5.x| Skip minor versions| No. You can't skip minor versions|